Examples/UIExplorer/NavigationExperimental/NavigationAnimatedExample.js

```diff
@@ -99,8 +100,10 @@ class NavigationAnimatedExample extends React.Component {
   _renderHeader(/*NavigationSceneRendererProps*/ props) {
     return (
       <NavigationHeader
-        {...props}
-        getTitle={state => state.key}
+        navigationProps={props}
```

Comment:
@hedgerwang We pass the same props to the renderer function, so splitting it means extra work and filtering out other props before passing it to the function. Here we just pass the same thing, so it' simpler.